From: James Troup <james@nocrew.org>
Subject: <varargs.h> is obsolete and no longer supported by gcc-3.3.
 ... and more to the point rlelib.c doesn't actually need it.

diff -urNad /home/james/debian/packages/xloadimage/xloadimage-4.1/rlelib.c xloadimage-4.1/rlelib.c
--- /home/james/debian/packages/xloadimage/xloadimage-4.1/rlelib.c	2003-06-06 03:32:44.000000000 +0100
+++ xloadimage-4.1/rlelib.c	2003-06-06 03:33:00.000000000 +0100
@@ -12,7 +12,6 @@
 
 #include <stdio.h>
 #include <math.h>
-#include <varargs.h>
 #include <ctype.h>
 
 #include "image.h"		/* need ZFILE definition */
